website/package.json

```diff
@@ -6,14 +6,14 @@
   "dependencies": {
     "@bugsnag/js": "^6.5.2",
     "@bugsnag/plugin-react": "^6.5.0",
-    "@hashicorp/nextjs-scripts": "^6.0.0-2",
+    "@hashicorp/nextjs-scripts": "^6.2.0-7",
```

Comment:
@jescalan are we able to get a non-prerelease on this PR?

Reply:
not quite yet unfortunately - we're still on prerelease for the css changes which this is based on top of. we can push those out as stable, but hashiconf and www are not on it yet. mike is working on getting them converted this week!